### PR TITLE
More brig/wanderer jobs

### PR DESCRIPTION
**Pull request by unjown**
**Content (Artwork / Missions / Jobs)**
Transport ~~captured unfettered spies~~ prisoners

## Summary
I added a brig / wanderer job, since there are lack of  jobs that require brig

## PR Checklist
 - [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [ ] I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}
 - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}

### CODE REVIEW

**Comment by EjoThims** [comment]
May be best to just go with "prisoners" outright. Wanderers seem like the type to rescue prisoners from ships when they can, but Unfettered don't really seem like the spying type, and they wouldn't be especially needed in this particular conflict either. Especially in the bulk enough to generate repeating jobs.

File: data/wanderer/wanderer jobs.txt [hunk 1]
```diff
@@ -839,3 +839,22 @@ mission "Wanderers to Kor Efreti"
...
+	source
+		has "language: Wanderer"
+		random < 20
+		government "Wanderer"
+	destination
+		distance 2 8
+		government "Wanderer"
+	on complete
+		payment 50000
+		dialog "You wish the Wanderers the best of luck on <planet> and collect your payment of <payment>."
```
Comment (roadrunner56) [inline data/wanderer/wanderer jobs.txt:860]: You're not carrying Wanderers, so this line makes no sense.
```suggestion
		dialog "Once you land on <planet>, you watch as some Wanderers in uniform carry the Unfettered prisoners off of your ship. You collect your payment of <payment>."
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]
I really like this PR, we need more variety in Wanderer space.

**Review by roadrunner56 — APPROVED** [review]

File: data/wanderer/wanderer jobs.txt [hunk 2]
```diff
@@ -839,3 +839,22 @@ mission "Wanderers to Kor Efreti"
...
+	job 
+	repeat
+	description "Transport <bunks> captured Unfettered soldiers to <planet>, payment is <payment>"
+	passengers 3
+	on offer
+		require "Brig"
+	source
+		has "language: Wanderer"
+		random < 20
+		government "Wanderer"
```
Comment (roadrunner56) [inline data/wanderer/wanderer jobs.txt:854]:
```suggestion
	source
		government "Wanderer"
```

File: data/wanderer/wanderer jobs.txt [hunk 3]
```diff
@@ -839,3 +839,22 @@ mission "Wanderers to Kor Efreti"
 	on complete
 		payment 35000 300
 		dialog `When you return, the Wanderers thank you for taking them to the Kor Efreti and hand you <payment> before disembarking.`
+
+mission "Captured soldiers"
+	name "Transport captured Unfettered soldiers"
+	job 
+	repeat
+	description "Transport <bunks> captured Unfettered soldiers to <planet>, payment is <payment>"
```
Comment (roadrunner56) [inline data/wanderer/wanderer jobs.txt:847]:
```suggestion
	description "Transport <bunks> captured Unfettered soldiers to <planet>, payment is <payment>"
	to offer
		has "language: Wanderer"
		random < 20
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]
I overlooked what I believe to be syntax errors in your code, that result in some github tests not passing. I'm admittedly not super experienced with ES mission syntax, so if someone else could double-check what I'm doing, that would be great.

**Review by roadrunner56 — APPROVED** [review]

**Comment by Galaucus** [comment]
This PR appears to be redundant with #6083.

**Pull request closed without merging**